tensorflow/lite/kernels/internal/common.h

```diff
@@ -138,31 +138,32 @@ inline void BiasAndClamp(float clamp_min, float clamp_max, int bias_size,
 #endif
 }
 
-inline int32_t MultiplyByQuantizedMultiplierSmallerThanOneExp(
-    int32_t x, int32_t quantized_multiplier, int left_shift) {
-  using gemmlowp::RoundingDivideByPOT;
-  using gemmlowp::SaturatingRoundingDoublingHighMul;
-  return RoundingDivideByPOT(
-      SaturatingRoundingDoublingHighMul(x, quantized_multiplier), -left_shift);
+inline int32_t MultiplyByQuantizedMultiplier(int32_t x,
+                                             int32_t quantized_multiplier,
+                                             int shift) {
+  assert(quantized_multiplier >= 0);
```

Comment:
instead of assert, please use TFLITE_DCHECK whenever possible, thanks

Reply:
Done, I also changed them in the int64 version as I initially used `assert` for coherence with this version.